tools/Dashboard/DevHome.Dashboard/ViewModels/WidgetViewModel.cs

```diff
@@ -183,6 +190,7 @@ private async Task RenderWidgetFrameworkElementAsync()
                     {
                         _renderedCard.Action += HandleAdaptiveAction;
                         WidgetFrameworkElement = _renderedCard.FrameworkElement;
+                        AnnounceWarnings(card.AdaptiveCard);

```

Comment:
Can we scope this to only when a screen reader is running? So we don't have to do a ton of tree traversals unnecessarily.
https://www.hanselman.com/blog/programmatically-detecting-screen-readers